.github/dependabot.yml

```diff
@@ -23,3 +33,19 @@ updates:
       - dependency-name: "jmespath"
       - dependency-name: "urllib3"
       - dependency-name: "wheel"
+
+  - package-ecosystem: "pip"
+    directory: "/"
+    open-pull-requests-limit: 10
+    schedule:
+      interval: "weekly"
```

Comment:
nit: the v2 schedule is daily, any reason this is weekly?

Reply:
Primarily to limit noise. Given we haven't merged any of these same day, it just saves on wasted runners. Happy to move to a higher cadence if we think we'll action on that.